pkg/cmd/openshift-apiserver/openshiftapiserver/config.go

```diff
@@ -163,7 +163,11 @@ func NewOpenshiftAPIConfig(config *openshiftcontrolplanev1.OpenShiftAPIServerCon
 		admission.DecoratorFunc(namespaceLabelDecorator.WithNamespaceLabelConditions),
 		admission.DecoratorFunc(admissionmetrics.WithControllerMetrics),
 	}
-	genericConfig.AdmissionControl, err = originadmission.NewAdmissionChains([]string{}, config.AdmissionPluginConfig, admissionInitializer, admissionDecorators)
+	expliticOn := []string{}
+	for plugin, _ := range config.AdmissionPluginConfig {
```

Comment:
Missing check in this block to see if it was explicitly turned off.  This may well cause you bugs in integration.

Reply:
> Missing check in this block to see if it was explicitly turned off. This may well cause you bugs in integration.

at the very least, open an issue.